### PR TITLE
[NO MERGE] DDL migrations in preparation of counting changes

### PR DESCRIPTION
**Pull request by marten**
* Makes sure looking up classifications by subject_id is an indexed query.
* Makes sure we can look up whether a project was live at time of classification using an indexed query.

# Review checklist

* First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
* If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
* If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
* Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: lib/tasks/migrate.rake [hunk 1]
```diff
@@ -117,6 +117,11 @@ namespace :migrate do
         .update_all(lifecycled_at: Time.current.to_s(:db))
       end
     end
+
+    desc "Fill live_project"
+    task fill_live_project: :environment do
+      Classification.where(live_project: nil).update_all("live_project = (metadata->>'live_project')::boolean")
```
Comment (marten) [inline lib/tasks/migrate.rake]: This one takes a bunch of minutes, and I'm not sure if it does a table lock (at least for writes). Any thoughts on how to do it better?

Comment (camallen) [inline lib/tasks/migrate.rake]: split it into small batches and update them, ala what you did [here](https://github.com/zooniverse/Panoptes/blob/04d6d77eed29b2e9ced5ff771707851bbb679f11/lib/tasks/migrate.rake#L80)

Comment (camallen) [inline lib/tasks/migrate.rake]: What is the use case for the `Classification.where(live_project: true)` query, in the dumps?

**Comment by marten** [comment]
After some discussions IRL, I'm going to rework this.

**Comment by marten** [comment]
I don't know why this is conflating multiple things anymore. Closing this PR and starting over.

**Pull request closed without merging**